src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -246,6 +224,57 @@ public void init() {
         initShowTrackingNotification();
     }
 
+    public void initDragAndDrop() {
+        Tab dndIndicator = new Tab(Localization.lang("Open files..."), null);
```

Comment:
That's a valid suggestion and I thought a bit about it. In the end, I prefer "open files" since it makes the action clearer for the user: she drops files into JabRef where they end up as new libraries.